### PR TITLE
Fix link to RDF-DATASET-NORMALIZATION 

### PR DESCRIPTION
**Pull request by cwebber**
(formerly RDF-DATASET-CANONICALIZATION).  Fixes one of the links from #2.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -589,7 +589,7 @@
           <li>
 Generate a <var>canonicalized document</var> by canonicalizing
 <var>document</var> according to a <a>canonicalization algorithm</a>
-(e.g. the <em>GCA2015</em> [[!RDF-DATASET-CANONICALIZATION]] algorithm).
+(e.g. the <em>GCA2015</em> [[!RDF-DATASET-NORMALIZATION]] algorithm).
```
Comment (dlongley) [inline index.html:592]: `GCA2015` => `URDNA2015`

File: index.html [hunk 2]
```diff
@@ -654,7 +654,7 @@
           <li>
 Generate a <var>canonicalized document</var> by canonicalizing
 <var>document</var> according to the <a>canonicalization algorithm</a>
-(e.g. the </var><em>GCA2015</em> [[!RDF-DATASET-CANONICALIZATION]] algorithm).
+(e.g. the </var><em>GCA2015</em> [[!RDF-DATASET-NORMALIZATION]] algorithm).
```
Comment (dlongley) [inline index.html:657]: `GCA2015` => `URDNA2015`

File: index.html [hunk 3]
```diff
@@ -709,7 +709,7 @@
               <li>
 Creating a <var>canonicalized options document</var> by canonicalizing
 <var>options</var> according to the <a>canonicalization algorithm</a>
-(e.g. the <em>GCA2015</em> [[!RDF-DATASET-CANONICALIZATION]] algorithm).
+(e.g. the <em>GCA2015</em> [[!RDF-DATASET-NORMALIZATION]] algorithm).
```
Comment (dlongley) [inline index.html:712]: `GCA2015` => `URDNA2015`

**Review by dlongley — CHANGES_REQUESTED** [review]
All instances of `GCA2015` should be `URDNA2015`.

This spec may have actually been using the term canonicalization on purpose because of a pending naming issue in the RDF Normalization spec where canonicalization is suggested -- and is likely to be approved. That being said, this spec should reflect the naming of *today*, not *tomorrow* (so links work, etc.) so this is a good PR.

**Comment by msporny** [comment]
> GCA2015 => URDNA2015

URDNA2015 is the current algorithm. It's being renamed to GCA2015 or DCA2015 to match the new naming. Fine with it being changed back, but I changed this on purpose to match new changes to the normalization spec being renamed to canonicalization spec. We should talk about this. Fine w/ @cwebber's changes being pulled in to fix the spec as it stands today.

**Pull request merged**